etc/pgbouncer.ini

```
@@ -161,6 +161,11 @@ auth_file = /etc/pgbouncer/userlist.txt
 ;; is off, server_reset_query is used only for session-pooling.
 ;server_reset_query_always = 0
 
+;; Comma-separated list of parameters to track per client. The Postgres 
+;; parameters listed here will be cached per client by pgbouncer and 
+;; restored in server everytime the client runs a query. 
+;track_startup_parameters = search_path
```

Let's use things that work on regular postgres in the docs.
```suggestion
;track_startup_parameters = IntervalStyle
```